src/features/discord/messages/interfaces/commands/flags/discord-command-flag-error.ts

```diff
@@ -0,0 +1,26 @@
+import { DiscordCommandFlagErrorTitleEnum } from "../../../enums/commands/flags/discord-command-flag-error-title.enum";
+
+export interface IDiscordCommandFlagError {
+  /**
+   * @description
+   * The description of the error
+   *
+   * Basically what is causing the error
+   * How to solve it
```

Comment:
Add a comment "Usually used as the value for an embed field" and use a link to the model.